velox/exec/OutputBufferManager.cpp

```diff
@@ -125,17 +125,25 @@ bool OutputBufferManager::updateOutputBuffers(
     const std::string& taskId,
     int numBuffers,
     bool noMoreBuffers) {
+  // If the buffer for the task is not found, then we don't need this update,
```

Comment:
Can we remove this comment?

I think the [declaration of this method](https://github.com/facebookincubator/velox/blob/0b4876fc55604c22e786040740aad4cb731842f3/velox/exec/OutputBufferManager.h#L31C1-L31C1) already specifies this logic

I checked other APIs, they also already comment it out in method declaration for example [getData](https://github.com/facebookincubator/velox/blob/0b4876fc55604c22e786040740aad4cb731842f3/velox/exec/OutputBufferManager.h#L65)